client-src/elements/chromedash-drawer.ts

```diff
@@ -177,13 +174,13 @@ export class ChromedashDrawer extends LitElement {
       fetch('/dev/mock_login', {method: 'POST'})
         .then(response => {
           if (!response.ok) {
-            throw new Error('Sign in failed! Response:', response);
+            throw new Error('Sign in failed! Response: ${response.status}');
           }
         })
         .then(() => {
           setTimeout(() => {
-            const url = window.location.href.split('?')[0];
-            window.location = url;
+            const url = window.location.href.split('?')[0] as string & Location;
```

Comment:
Try to avoid `as` clauses when you can, since the typechecker doesn't check them. I think you just want

```suggestion
            const url = window.location.href.split('?')[0]!;
```
here.